src/Symfony/Component/AssetMapper/Command/ImportMapRequireCommand.php

```diff
@@ -44,6 +44,7 @@ protected function configure(): void
             ->addArgument('packages', InputArgument::IS_ARRAY | InputArgument::REQUIRED, 'The packages to add')
             ->addOption('entrypoint', null, InputOption::VALUE_NONE, 'Make the package(s) an entrypoint?')
             ->addOption('path', null, InputOption::VALUE_REQUIRED, 'The local path where the package lives relative to the project root')
+            ->addOption('resolver', null, InputOption::VALUE_REQUIRED, 'The alias of the package resolver to use')
```

Comment:
This, _imho_ cannot offer a predictable DX as importmap:install / importmap:require / importmap:update (etc..) would then be inconsistent.

Reply:
I think it won't have an impact as the resolver config is added to the importmap file.
During `(importmap:install / importmap:update)` commands, if a custom resolver was provided, it will be used, otherwise the default one (jsdeliver) will be used.
It will just be a matter of editing `importmap.php`